packages/ui/src/components/Channel/Scroller.tsx

```diff
@@ -162,11 +162,14 @@ export default function Scroller({
     [anchor, hasFoundAnchor]
   );
 
+  const theme = useTheme();
+
   // Used to hide the scroller until we've found the anchor post.
   const style = useMemo(() => {
     return {
       opacity: hasFoundAnchor ? 1 : 0,
-      backgroundColor: 'white',
+      backgroundColor: theme.background.val,
+      paddingTop: channelType === 'gallery' ? 8 : 0,
```

Comment:
Could this be applied by the component wrapping this? Either way a comment explaining what we're compensating for here would be helpful.

Reply:
Are you referring to the color or the padding? I set the color to just be whatever the theme's background is because setting it to `white` gives a white background when the app is in dark mode.

The padding could be set elsewhere, yeah.

Reply:
Yep the background makes sense, was just looking at the padding.

Reply:
That padding was totally extraneous afaict, just removed it.